src/System.ServiceModel.Federation/src/System/ServiceModel/Federation/WSTrustChannelSecurityTokenProvider.cs

```diff
@@ -27,7 +24,8 @@ namespace System.ServiceModel.Federation
 {
 
     /// <summary>
-    /// Custom WSTrustChannelSecurityTokenProvider that returns a SAML assertion
+    /// <see cref="WSTrustChannelSecurityTokenProvider"/> has been designed to work with <see cref="WSFederationHttpBinding"/> to that will send a WsTrust message to obtain a token from an STS and add the token as
+    /// an issued token when communicating with a WCF relying party.
```

Comment:
I think you merged two different phrases in your head. "to that will send" doesn't make sense.

Reply:
fixed in: b160df7f33caf6447f14e47b81ab3ff9f5930924